Editor/TextProvider.cs

```diff
@@ -182,6 +182,33 @@ internal sealed class TextProvider
             { Strings.TabAnywhere, "GameLift Anywhere" },
             { Strings.TabEC2, "Managed EC2" },
             { Strings.TabHelp, "Learning Resources" },
+            
+            { Strings.LabelAccountTitle, "Manage Your User Profiles"},
```

Comment:
Could you please add screenshots for these views. That would make reviewing much easier.

Reply:
Screenshots provided below in a comment.